docs/about/github_stars.py

```diff
@@ -0,0 +1,147 @@
+from bokeh.plotting import figure, show
+import datetime
+from dateutil import relativedelta
+from dateutil.parser import isoparse
+import os
+
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+
+_SECRET_ENV_VAR = "GITHUB_GRAPHQL"
```

Comment:
Where did you get this from? Do we have an account or something?

Reply:
It's using a token from my account. Set with only a few privileges, but not ideal.

Reply:
Ooh... this seems like a very opaque dependence, but I might not understand this right.